example/lib/main.dart

```diff
@@ -2,24 +2,24 @@ import 'package:flutter/foundation.dart';
 import 'package:flutter/material.dart';
 import 'dart:async';
 import 'dart:io' show Platform;
-
+import 'package:fluttertoast/fluttertoast.dart';
 import 'package:apptentive_flutter/apptentive_flutter.dart';
 
-import 'package:firebase_core/firebase_core.dart';
-import 'package:firebase_messaging/firebase_messaging.dart';
+// import 'package:firebase_core/firebase_core.dart';
+// import 'package:firebase_messaging/firebase_messaging.dart';
 
-Future<void> _firebaseMessagingBackgroundHandler(RemoteMessage message) async {
-  print("Apptentive: handling a background message");
-  print("Notification Data: ${message.data}");
-}
+// Future<void> _firebaseMessagingBackgroundHandler(RemoteMessage message) async {
+//   print("Apptentive: handling a background message");
+//   print("Notification Data: ${message.data}");
+// }
 
 String? integration_token = "";
 
```

Comment:
Example app is fixed. I couldn't work on push. The rest of the interactions should work.